dts/bindings/riscv/riscv,cpus.yaml

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2021 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Import from devicetree/bindings/riscv/cpus.yaml
+
+include: cpu.yaml
+
+description: |
+  This document uses some terminology common to the RISC-V community
+  that is not widely used, the definitions of which are listed here:
+
+  hart: A hardware execution context, which contains all the state
+  mandated by the RISC-V ISA: a PC and some registers.  This
+  terminology is designed to disambiguate software's view of execution
+  contexts from any particular microarchitectural implementation
+  strategy.  For example, an Intel laptop containing one socket with
+  two cores, each of which has two hyperthreads, could be described as
+  having four harts.
```

Comment:
This was copied word by word from a GPL-licensed code/documentation. I don't think that is legal.

https://github.com/riscv/riscv-device-tree-doc/blob/master/bindings/riscv/cpus.txt
https://github.com/riscv/riscv-device-tree-doc/blob/master/GPL-2.0.license